relay-server/src/actors/processor/span.rs

```diff
@@ -43,376 +24,395 @@ pub fn filter(state: &mut ProcessEnvelopeState) {
     });
 }
 
+/// Contains the processing-only functionality.
 #[cfg(feature = "processing")]
-pub fn process(state: &mut ProcessEnvelopeState, config: Arc<Config>) {
-    use relay_event_normalization::RemoveOtherProcessor;
-
-    let span_metrics_extraction_config = match state.project_state.config.metric_extraction {
-        ErrorBoundary::Ok(ref config) if config.is_enabled() => Some(config),
-        _ => None,
-    };
-
-    let config = NormalizeSpanConfig {
-        received_at: state.managed_envelope.received_at(),
-        transaction_range: AggregatorConfig::from(
-            config.aggregator_config_for(MetricNamespace::Transactions),
-        )
-        .timestamp_range(),
-        max_tag_value_size: config
-            .aggregator_config_for(MetricNamespace::Spans)
-            .max_tag_value_length,
-    };
-
-    state.managed_envelope.retain_items(|item| {
-        let mut annotated_span = match item.ty() {
-            ItemType::OtelSpan => {
-                match serde_json::from_slice::<relay_spans::OtelSpan>(&item.payload()) {
-                    Ok(otel_span) => Annotated::new(otel_span.into()),
+mod processing {
```

Comment:
Could also make this a real module in `span/processing.rs` without moving `span.rs`